src/runner/browser-runner.ts

```diff
@@ -59,6 +59,10 @@ export class BrowserRunner extends Runner {
         return true;
     }
 
+    async waitTestsCompletion(): Promise<void> {
```

Comment:
Implement new method in order to wait when all tests inside browsers will be completed.